db/new_seeds/base/add_schools_and_local_authorities.rb

```diff
@@ -27,7 +27,8 @@ def add_school_to_local_authority(school:, local_authority:, lead_providers:, co
       scenarios.flat_map(&:mentees).map(&:participant_profile).each do |participant_profile|
         Rails.logger.debug("seeding eligibility for #{participant_profile.user.full_name}")
 
-        FactoryBot.create(:seed_ecf_participant_eligibilty, random_weighted_eligibility_trait, participant_profile:)
+        participant_profile.ecf_participant_eligibility&.destroy!
+        FactoryBot.create(:seed_ecf_participant_eligibility, random_weighted_eligibility_trait, participant_profile:)
```

Comment:
How come we're destroying the eligibility first? I thought unless it had been explicitly created it shouldn't be there? 🤔

Reply:
Yes, not elegant, I know. The thing is that now those mentees have eligibility when they are created because the scenario adds it. Otherwise ECTs and mentors not get properly displayed and categorised in the dashboard.

I thought this dirty approach of deleting it and then recreate it with a random status would be better than passing parameters to the scenario to say "Do not create eligibility for these ECTs". That would also make the scenario code more complicated. 

Happy to rethink this if we are not happy even though this is just seeding code.